styles/main_styles.css

```diff
@@ -96,7 +96,7 @@ h1 {
 .connect {
   font-style: normal;
   font-weight: 500;
-  color: #7f8cff;
+  color: #606bcc;
```

Comment:
- [ ] **Required:** A minimum website contrast of 4.5:1 is required to pass this test. At the moment your score is 4.4:1, with a bit more color contrast, you should pass this benchmark. [Check this resource out.](https://www.boia.org/blog/the-5-most-common-website-accessibility-issues-and-how-to-fix-them)

Reply:
Actually, on my device it shows a value of 4.7:1 instead of 4.4:1.
Regardless I would increase the value and re-submit